.github/workflows/php-coding-standards.yml

```diff
@@ -0,0 +1,44 @@
+name: PHP Coding Standards
+
+on:
+  push:
+    branches: [ trunk ]
+  pull_request:
+    branches: [ trunk ]
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        php: [ 8.2 ]
```

Comment:
php 8.2 is reaching end of active support should we use 8.3 here?
<img width="641" alt="image" src="https://github.com/user-attachments/assets/d98e44a9-2f72-4c31-9a1a-1645b92aaa0c">

Reply:
Done!